
From: Nick Piggin <nickpiggin@yahoo.com.au>

The "[PATCH 5/8] sched add load balance flag" patch forgot to update ia64's
definition of SD_NODE_INIT to use the SD_LOAD_BALANCE flag, causing it to not
do any periodic load balancing across nodes.  Update it.

Signed-off-by: Nick Piggin <nickpiggin@yahoo.com.au>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-ia64/processor.h |    3 ++-
 1 files changed, 2 insertions(+), 1 deletion(-)

diff -puN include/asm-ia64/processor.h~sched-sched-add-load-balance-flag-fix include/asm-ia64/processor.h
--- 25/include/asm-ia64/processor.h~sched-sched-add-load-balance-flag-fix	Tue Sep 14 18:43:49 2004
+++ 25-akpm/include/asm-ia64/processor.h	Tue Sep 14 18:43:49 2004
@@ -346,7 +346,8 @@ struct task_struct;
 	.cache_hot_time		= (10*1000000),		\
 	.cache_nice_tries	= 1,			\
 	.per_cpu_gain		= 100,			\
-	.flags			= SD_BALANCE_EXEC	\
+	.flags			= SD_LOAD_BALANCE	\
+				| SD_BALANCE_EXEC	\
 				| SD_WAKE_BALANCE,	\
 	.last_balance		= jiffies,		\
 	.balance_interval	= 10,			\
_
